pycheribuild/projects/cross/cheribsd.py

```diff
@@ -319,7 +319,7 @@ def default_kernel_config(self):
                         return "CHERI-CAPREVOKE-QEMU-FETT"
                     return "CHERI-CAPREVOKE-QEMU"
                 if self.purecap_kernel:
-                    return "CHERI-PURECAP-QEMU-NODEBUG"
+                    return "CHERI-PURECAP-QEMU"
```

Comment:
Please revert this; we don't want everyone to be waiting twice as long for QEMU to boot...